frontend/src/component/common/DialogFormTemplate/DialogFormTemplate.tsx

```diff
@@ -15,6 +15,11 @@ import {
 import { Button } from '@mui/material';
 import { CreateButton } from 'component/common/CreateButton/CreateButton';
 import type { IPermissionButtonProps } from 'component/common/PermissionButton/PermissionButton';
+import type { FeatureNamingType } from 'interfaces/project';
+import { ConditionallyRender } from '../ConditionallyRender/ConditionallyRender';
+import { NamingPatternInfo } from './NamingPatternInfo';
+
+type NamingPattern = FeatureNamingType;
```

Comment:
Just aliasing the type here. We don't have naming patterns for anything but features, but there is no reason why we couldn't, and there's nothing in the type that relates to features specifically, so this is just to make it more "agnostic". Happy to remove the aliasing if we don't like it.

Reply:
Are you trying to do 

`
import { NamingPatternInfo as NamingPattern } from './NamingPatternInfo';`

Reply:
No, no. NamingPatternInfo is a component. NamingPattern is a type.